content/sql/concepts/commands/terms/select/select.md

```diff
@@ -16,36 +16,36 @@ CatalogContent:
   - 'paths/design-databases-with-postgresql'
 ---
 
-Fetchs data from a database. Every query will begin with `SELECT`.
+Every SQL query will begin with the `SELECT` command to fetch data from one or more tables.
 
 ## Syntax
 
-The `SELECT *` statement returns all columns from the provided table in the result set:
+The `SELECT` command is followed by one or more of column names to be returned from the table(s) listed in the `FROM` clause:
 
 ```sql
-SELECT *
-FROM table_name;
+SELECT column1, column2, ... columnN FROM table;
 ```
 
-You can also select individual columns:
+`SELECT` can also use the special `*` character to represent all columns from the table(s):
 
 ```sql
-SELECT column_name(s)
-FROM table_name;
+SELECT * FROM table;
 ```
 
+**Note:** It is best practice to never rely on column order when using `*`.
+
 ## Examples
 
-The following query will fetch all columns and records (rows) from the `movies` table:
+The following example selects all columns from `table1` and `table2` with an [`INNER JOIN`](https://www.codecademy.com/resources/docs/sql/joins) restricting the rows to where `table1.columnA` and `table2.columnB` are equal:
 
 ```sql
-SELECT *
-FROM movies;
+SELECT * FROM table1 INNER JOIN table2 ON table1.columnA = table2.columnB;
 ```
 
-And this query will select just the `title` and `rating` columns from the `movies` table:
+When selecting all columns from across multiple tables, duplicate column names will generate an error unless they are distinguished with an alias.
+
+The next example selects all columns from `table1` and a named column `columnA` from `table2`. It is presumed that both tables have a column named `columnA` and, therefore, an alias should be used to avoid an error :
```

Comment:
```suggestion
The next example selects all columns from `table1` and a named column, `columnA`, from `table2`. It is presumed that both tables have a column named `columnA` and, therefore, an alias should be used to avoid an error:
```